lib/operations/collection_ops.js

```diff
@@ -156,14 +156,17 @@ function bulkWrite(coll, operations, options, callback) {
 
 // Check the update operation to ensure it has atomic operators.
 function checkForAtomicOperators(update) {
-  const keys = Object.keys(update);
+  const keys = Array.isArray(update)
```

Comment:
Is it ok to be modifying `update` here? Also, is there a reason that `Array.isArray` is being checked in here as opposed to in `updateOne`?

Reply:
ohh ok makes sense! 👍